dask_expr/_collection.py

```diff
@@ -1109,6 +1109,20 @@ def iloc(self):
 
         return ILocIndexer(self)
 
+    def nunique(self, axis=0, dropna=True):
+        if axis == 1:
+            return new_collection(
+                expr.NUniquePerBlock(self.expr, axis=axis, dropna=dropna)
+            )
+        else:
+            return concat(
+                [
+                    col.nunique(dropna=dropna).to_series(col.name)
+                    # for col in self  # TODO not implemented
+                    for col in (self.iloc[:, i] for i in range(self.shape[1]))
```

Comment:
Can you write this as

```
self.iloc[:, I].(dropna=dropna).to_series(col.name) for i in range(len(self.columns))
```
?

Reply:
Could you merge https://github.com/dask-contrib/dask-expr/pull/480?

Reply:
done